Team_Harrison_FaultSeeded.cpp

```diff
@@ -343,8 +329,6 @@ void Student::readFile(string filepath) {
         studentList.push_back(new Student(n, u, e, pres, es, proj));
```

Comment:
Consider swapping the order the grades are put into the call, such as new Student(n, u, e, proj,pres,es)). Would be a difficult bug to catch.

Reply:
I like your readFile fault suggestion.
Also, I'm not sure if that is necessary, though it wouldn't hurt to implement that. Should be an easy fix in a couple places